### ISSUE TITLE
Support for this premade board?

### ISSUE THREAD

**Issue by oleost** [n0bel/ESPrinkler2#2]
Hi.

Any chance this could work on this premade board?

"Esp8266 4ch 12v wifi relay"
https://www.aliexpress.com/item/ESP8266-4CH-12V-WiFi-Relay-Module-Smart-Home-Mobile-APP-Remote-Control-Switch/32879651251.html?spm=2114.search0104.3.1.268b39e3txTFgh&ws_ab_test=searchweb0_0,searchweb201602_1_10152_10065_10151_10344_10068_10342_10343_10340_10059_10341_5012011_10696_100031_5011911_10084_10083_10103_10618_10307_10624_10623_10622_10621_10620,searchweb201603_54,ppcSwitch_2&algo_expid=5b4c92ad-dcd2-4901-8fda-b2b61f445251-0&algo_pvid=5b4c92ad-dcd2-4901-8fda-b2b61f445251&transAbTest=ae803_2&priceBeautifyAB=0

Without oled support ofc, and only limited to 4 outputs.

**Comment by theevilshiftkey** [comment]
I don't think that the ESP01 has enough memory for this project. If I have time (not likely) I may fork this to work on some premade board not yet researched, since I only need one zone/output.
